pod/pod-api-public.yaml

```diff
@@ -6688,6 +6723,21 @@ definitions:
       apiKey:
         type: string
         description: "apiKey sent into every callback request, using the X-API-KEY header"
+  AppProperties:
+    type: array
+    description: Application configuration properties that are shared with the extension application, client side. Do not store sensitive information here.
+    items:
+      $ref: '#/definitions/AppProperty'
+  AppProperty:
+    type: object
+    description: Application configuration property that is shared with the extension application, client side. Do not store sensitive information here.
+    properties:
+      key:
+        type: string
+        description: Name of an application configuration property. It cannot be null or empty and its length is limited to 1014 characters.
```

Comment:
The errors here are fixed by https://github.com/SymphonyOSF/SBE/pull/23896 (number of chars)